src/main/kotlin/src/step2/Operation.kt

```diff
@@ -0,0 +1,67 @@
+package src.step2
+
+import kotlin.IllegalArgumentException
+
+interface Calculable {
+    fun calculate(x: Int, y: Int): Int
+}
+
+enum class Operation(val symbol: Char) : Calculable {
+    PLUS('+') {
+        override fun calculate(x: Int, y: Int): Int {
+            return x + y
+        }
+    },
+    MINUS('-') {
+        override fun calculate(x: Int, y: Int): Int {
+            return x - y
+        }
+    },
+    MULTIPLY('*') {
+        override fun calculate(x: Int, y: Int): Int {
+            return x * y
+        }
+    },
+    DIVIDE('/') {
+        override fun calculate(x: Int, y: Int): Int {
+            return x / y
+        }
+    }
+}
+
+object StringCalculator {
+
+    fun execute(value: String?): Int {
+        validate(value)
+
+        var result = 0
+        var symbol: Char? = null
+        value!!.forEachIndexed forEach@{ index, it ->
+            // 연산자를 저장
+            if (!it.isDigit()) {
+                symbol = it
+                return@forEach
+            }
+
+            // 첫번째 숫자값을 저장
+            if (index == 0) {
+                result = Character.getNumericValue(it)
+            }
+
+            // 계산
+            val operation = Operation.values().find { it.symbol == symbol }
+            result = operation?.calculate(result, Character.getNumericValue(it))
```

Comment:
```suggestion
            result = operation.calculate(result, Character.getNumericValue(it))
```
opration에 대해 null이 아님을 보장하려면 어떻게 해야 할까요?